### PR TITLE
Add bionic manage and remove useless test

### PR DESCRIPTION
**Pull request by cdelgehier**

### CODE REVIEW

File: checks.bats [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
-    [[ "${output}" =~ "/etc/apt/sources.list.d/openio-sds-17.04.list /etc/apt/sources.list.d/openio-test-17.04.list" ]]
-  elif [ "${DISTRIBUTION}" == "ubuntu" ]; then
-    run docker exec -ti ${SUT_ID} bash -c 'ls /etc/apt/sources.list.d/openio* | tr "\n" " "'
-    echo "output: "$output
-    echo "status: "$status
-    [[ "${status}" -eq "0" ]]
-    [[ "${output}" =~ "/etc/apt/sources.list.d/openio-sds-16.10.list /etc/apt/sources.list.d/openio-sds-17.04.list /etc/apt/sources.list.d/openio-test-17.04.list" ]]
-  fi
-}
+#@test 'Check repository files for multiple product distribution' {
```
Comment (vincent-legoll) [inline checks.bats:7]: Why commenting instead of removing ? Is this coming back soon ?

File: test.yml [hunk 2]
```diff
@@ -7,17 +7,27 @@
...
+      openio_repository_openstack_release: queens
       openio_repository_products:
         sds:
-          release: "16.10"
-      when: ansible_distribution != 'Debian'
+          release: '18.04'
+      when: ansible_distribution_release == 'bionic'
 
+
+##    - role: role_under_test
```
Comment (vincent-legoll) [inline test.yml:20]: Ditto, why is that commented code here ? will it be activated in the future, or is this just sample code (a comment would be welcome then) ?

File: test.yml [hunk 3]
```diff
@@ -7,17 +7,27 @@
...
+##      openio_repository_products:
+##        sds:
+##          release: "16.10"
+##      when: ansible_distribution != 'Debian'
+#
     - role: role_under_test
+      openio_repository_mirror_host: mirror2.openio.io
```
Comment (vincent-legoll) [inline test.yml:27]: Why is this only on this role inclusion and not the other ones above ? Do we want to test both mirrors (a comment saying so would also be welcome, then) ?

**Comment by racciari** [comment]
Any update here ?

**Pull request closed without merging**